package.json

```diff
@@ -17,10 +17,10 @@
         "db:revert": "./node_modules/.bin/typeorm migration:revert --config ./lib/src/ormconfig.js ",
         "integration-test": "mocha --require source-map-support/register --require make-promises-safe 'lib/integration-test/**/*_test.js' --timeout 200000 --exit",
         "test": "yarn test:rest && yarn test:metatx && yarn test:rfqt && yarn test:rfqm",
-        "test:metatx": "env-cmd -f ./test/test_env mocha --require source-map-support/register --require make-promises-safe 'lib/test/meta_transaction_**.js' --timeout 200000 --exit --bail",
+        "test:metatx": "env-cmd -f ./test/test_env mocha --require source-map-support/register --require make-promises-safe 'lib/test/meta_transaction_*.js' --timeout 200000 --exit --bail",
         "test:rest": "env-cmd -f ./test/test_env mocha --exclude lib/test/rfqt_test.js --exclude lib/test/meta_transaction_test.js --require source-map-support/register --require make-promises-safe 'lib/test/**/*_test.js' --timeout 200000 --exit --bail",
```

Comment:
Is it possible to also `--exclude lib/test/**/rfqm_*.js`? I think certain `rfqm` tests will get run twice if not excluded (once by `test:rest` and again by `test:rfqm`)

Reply:
done